core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/BaseResourceUpdateRequest.java

```diff
@@ -311,18 +318,17 @@ private List<String> getBundleAncestorIds(final TransactionContext context, fina
 			return List.of(bundleId);
 		}
 		
-		final Bundles bundles = ResourceRequests.bundles()
-			.prepareSearch()
-			.filterById(bundleId)
-			.one()
-			.build()
-			.execute(context);
+		final Resources bundles = ResourceRequests.prepareSearchCollections()
```

Comment:
This makes me confused again. So a Bundle is a specialized collection, sort of. But it is somewhat weird to type prepareSearch**Collection** to get the **Bundle**.

Reply:
Agree. Will take care of it next week.